projects/admin/src/app/circulation/patron/patron-transactions/patron-transaction-event/patron-transaction-event.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { PatronTransactionEvent, PatronTransactionEventType } from '../../../patron-transaction';
+import { OrganisationService } from '../../../../service/organisation.service';
+
+@Component({
+  selector: 'admin-patron-transaction-event',
+  templateUrl: './patron-transaction-event.component.html'
+})
+export class PatronTransactionEventComponent {
+
+  @Input() event: PatronTransactionEvent;
+  isCollapsed = true;
+  public patronTransactionEventType = PatronTransactionEventType;  // store a reference to enum to use in html template
```

Comment:
_public_ is not necessary and you can add some comments to explain the properties.

Reply:
done